samples/internat/internat.cpp

```diff
@@ -320,6 +320,12 @@ MyFrame::MyFrame()
     macro_menu->Append(INTERNAT_MACRO_8, wxGETTEXT_IN_CONTEXT_PLURAL("context_2", "sing", "plur", 1));
     macro_menu->Append(INTERNAT_MACRO_9, wxGETTEXT_IN_CONTEXT_PLURAL("context_2", "sing", "plur", 2));
 
+    // Also wide strings can be used:
```

Comment:
Shouldn't this rather go into `tests/allheaders.cpp` as the other checks you added there?

Or, perhaps, move all of these checks to `tests/intl/intltest.cpp`?

Reply:
> Shouldn't this rather go into `tests/allheaders.cpp` as the other checks you added there?

`allheaders.cpp` is special, because it defines `wxNO_IMPLICIT_WXSTRING_ENCODING`, and therefore doesn't test the regular use case. I also hypothesized that wide strings are inherently incompatible with `wxASCII_STR`, which the wrappers call in that case, but didn't test what actually happens.

> Or, perhaps, move all of these checks to `tests/intl/intltest.cpp`?

Yes, the new ones I added to the sample can be moved.

Reply:
When you put it that way, yes. I suppose this PR is complete now, after your latest changes.